app/services/public_transportation/import_service.rb

```diff
@@ -108,6 +115,10 @@ def parse_calendar_dates(zip_file)
     def parse_calendar(zip_file)
       p "Parsing calendar.txt"
       calendar_counter = 0
+
+      # cleanup stop_times bevor import
+      @redis.delete_all_gtfs_calendar(@data_provider_id)
```

Comment:
cleanup stop times and then calling deletion of calendar dates? what about `# cleanup before import` as comment without specifying?